completion_aggregator/signals.py

```diff
@@ -22,15 +22,15 @@ def register():
     """
     post_save.connect(completion_updated_handler, sender=compat.get_aggregated_model())
 
-    try:
+    try:  # pragma: no cover
```

Comment:
nit: would prefer to leave these out of the main code base, so we know they should be covered at some point.

Totally ok to have them in `compat.py` though, since its purpose is to isolate the integration layer with edx-platform.